rust/docs/OpenConsumerCreateRequestDto.md

```
@@ -4,14 +4,14 @@
 
 Name | Type | Description | Notes
 ------------ | ------------- | ------------- | -------------
-**app_id** | Option<**String**> | 第三方应用ID | [optional]
+**app_id** | **String** | 第三方应用ID |
```

markdownlint-cli2 (0.22.1)

[warning] Table pipe style
Expected: no_leading_or_trailing; Actual: trailing_only; Unexpected trailing pipe

(MD055, table-pipe-style)